src/store/modules/wallet.js

```diff
@@ -539,8 +573,15 @@ const actions = {
       return false;
     });
 
-    commit(WALLET_SET_FOLLOWEE_EVENTS, filteredEvents);
-    commit(WALLET_SET_FOLLOWEE_EVENT_FETCHING, false);
+    if (filteredEvents?.length) {
+      const existingEvents = state.followeeEvents || [];
+      const updatedEvents = [...existingEvents, ...filteredEvents];
+      commit(WALLET_SET_FOLLOWEE_EVENTS, updatedEvents);
+      if (!state.walletIsFetchingFolloweeEvents) {
```

Comment:
isn't `walletIsFetchingFolloweeEvents` a getter? also since we can have multiple events stuck in `processEvents()`, should we unset `WALLET_SET_FOLLOWEE_EVENT_FETCHING` in `fetchFolloweeWalletEvent` instead? 
Consider leaving `walletIsFetchingFolloweeEvents` as is and modify UI logic to just show any feed item fetched, before we check `walletIsFetchingFolloweeEvents` to show loading